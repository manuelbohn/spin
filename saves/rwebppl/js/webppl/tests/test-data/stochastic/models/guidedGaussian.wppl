var model = function() {
  var a = sample(Gaussian({mu: -4, sigma: 2}), {guide() {
    return Gaussian({
      mu: param(),
      sigma: Math.exp(param())
    });
  }});
  return a;
};
