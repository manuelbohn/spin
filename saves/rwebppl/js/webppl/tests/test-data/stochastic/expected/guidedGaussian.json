{
  "mean": -4,
  "std": 2
}
